docusearch/tests/test_views.py

```diff
@@ -1,17 +1,53 @@
 from django.core.urlresolvers import reverse
 from django.test import TestCase
+from django.http import QueryDict
 
 from docusearch.templatetags.get_filename import get_filename
+from docusearch.views import CustomSearchView
 
 
 class TestViews(TestCase):
 
+    fixtures = ['documents.json']
+
     def test_home_page(self):
         """ The /document/search/ page should load without errors. """
         response = self.client.get(reverse('home'))
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, "search and find documents released")
 
+    def test_search(self):
+        """ Document search should work properly """
+
+        # Empty search returns nothing, but won't fail
+        response = self.client.get('/documents/search/?q=')
```

Comment:
you should probably use reverse() to get the URL here, so we're not hard coding it.